types/config.d.ts

```diff
@@ -103,17 +101,6 @@ declare module Config {
     gatherers?: GathererJson[];
   }
 
-  interface NavigationJson extends SharedPassNavigationJson {
-    /** The identifier for the navigation. Config extension will deduplicate navigations with the same id. */
-    id: string;
-    /** Whether throttling settings should be skipped for the pass. */
-    disableThrottling?: boolean;
-    /** Whether storage clearing (service workers, cache storage) should be skipped for the pass. A run-wide setting of `true` takes precedence over this value. */
-    disableStorageReset?: boolean;
-    /** The array of artifacts to collect during the navigation. */
-    artifacts?: Array<string>;
```

Comment:
I didn't add any of these settings to `config.settings` because:
- `disableStorageReset` is already on `config.settings`
- `id` doesn't make sense if there is only one navigation
- `disableThrottling` can be achieved with `config.settings.throttlingMethod = 'provided'`
- `config.artifacts` is now the canonical list of artifacts, we don't need to maintain a second one